### PR TITLE
Charts - replace getResizeObserver

### PR DESCRIPTION
**Pull request by dlabrecq**
Replaced react-chart `getResizeObserver` with same function from react-core.

Related to PR https://github.com/patternfly/patternfly-react/pull/8533

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]

File: packages/pf-codemods/README.md [hunk 1]
```diff
@@ -324,3 +324,21 @@ Out:
 ```jsx
 <Tooltip     />
 ```
+
+### chart-resizeObserver-function [(#8533)](https://github.com/patternfly/patternfly-react/pull/8533)
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:328]: Nit: can this be placed alphabetically in the README

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
If there's a likelihood that react-charts' getResizeObserver was being imported by anyone else, and since the README includes an "Examples" section for in and output, it might be worth including a codemod that can be ran to fix the import from `'@patternfly/react-charts'` to `@patternfly/react-core'`. 

Alternatively we could just mention the change and remove the Examples section in the README. @gitdallas wdyt?

**Comment by gitdallas** [comment]
@thatblindgeye - i believe nicole just let dan know he should either create a story or at least update the readme for this. maybe one of us could take over creating the codemod for this - unless you want to, @dlabrecq

File: packages/pf-codemods/README.md (hunk 1 above)
Comment (dlabrecq) [reply packages/pf-codemods/README.md]: Done

**Comment by dlabrecq** [comment]
Per Nicole's instructions, I created issue https://github.com/patternfly/patternfly-react/pull/8533 and updated the README in this repo. I wasn't planning to create and test codemods, so I'm fine having someone else implement that.

**Review by thatblindgeye — COMMENTED** [review]
Sounds good! @gitdallas unless you'd want to add a codemod and tests for this, I could push those changes to this PR

**Comment by thatblindgeye** [comment]
@gitdallas let me know if you think the codemod should be simpler and just update the import path. Tried taking into account other scenarios with what I pushed.

File: packages/eslint-plugin-pf-codemods/lib/rules/v5/charts-resizeObserver-import.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                        ? range[1] + 1
+                        : range[1];
+
+                    fixes.push(
+                      fixer.replaceTextRange([range[0], rangeEnd], "")
+                    );
+                    fixes.push(
+                      fixer.insertTextAfterRange(
+                        node.range,
+                        "import {getResizeObserver} from '@patternfly/react-core';"
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/lib/rules/v5/charts-resizeObserver-import.js:38]: should we sneak a `\n` in before "import"?

**Comment by gitdallas** [comment]
thanks, lgtm. while you resolve - this would be a good opportunity to add to the `test.tsx` file btw and see the output with `yarn test:single`

**Review by gitdallas — APPROVED** [review]

**Comment by thatblindgeye** [comment]
@gitdallas tried out the test.tsx file, pretty neat! Wasn't sure if we should keep additions saved as part of PRs or if it's better just to try locally so I didn't include it here.

**Comment by dlabrecq** [comment]
@thatblindgeye Thank you for adding the codemods.

**Pull request merged**